update_prevalence.py

```diff
@@ -445,6 +447,7 @@ def rollup_testing(parent: Place, child_attr: str) -> None:
                             # These just don't have any reported cases
                             "Hoonah-Angoon, Alaska, US",
                             "Lake and Peninsula, Alaska, US",
+                            "Yakutat plus Hoonah-Angoon, Alaska, US",
```

Comment:
Huh, I wonder if this another "Bristol Bay plus Lake Peninsula" where the population is aggregated but the case data isn't. You can probably deal with it by adding an analogous special case around line 616.

Reply:
Okay. Fixed with a Bristol Bay-like solution.